src/lib/addons/webhook.ts

```diff
@@ -9,6 +9,7 @@ interface IParameters {
     bodyTemplate?: string;
     contentType?: string;
     authorization?: string;
+    customHeaders?: string;
```

Comment:
And one more... although this could be a bit trickier because we need to handle a web request.